stage8/00-license-generation/00-run.sh

```diff
@@ -261,14 +261,63 @@ echo "<th>Location</th>" >> ${FILE}
+wget https://raw.githubusercontent.com/analogdevicesinc/hdl/main/LICENSE
+html_pre_file LICENSE hdl
+rm LICENSE
+
+package_table_items $((var++)) "libiio" " " "LGPL-2.1/GPL-2.0" "https://github.com/analogdevicesinc/libiio"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/libiio/COPYING.txt libiio
+
+package_table_items $((var++)) "iio-oscilloscope" " " "GPL-2.0" "https://github.com/analogdevicesinc/iio-oscilloscope"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/iio-oscilloscope/LICENSE iio-oscilloscope
+
+package_table_items $((var++)) "scopy" " " "GPL-3.0" "https://github.com/analogdevicesinc/scopy"
+wget https://raw.githubusercontent.com/analogdevicesinc/scopy/main/LICENSE
+html_pre_file LICENSE scopy
+rm LICENSE
+
 package_table_items $((var++)) "libad9361-iio" " " "GPL-2.0" "https://github.com/analogdevicesinc/libad9361-iio"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/libad9361-iio/LICENSE libad9361-iio
+
 package_table_items $((var++)) "libad9166-iio" " " "GPL-2.0" "https://github.com/analogdevicesinc/libad9166-iio"
-package_table_items $((var++)) "fru-tools" " " "GPL-2.0" "https://github.com/analogdevicesinc/fru-tools"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/libad9166-iio/LICENSE libad9166-iio
+
+package_table_items $((var++)) "fru_tools" " " "GPL-2.0" "https://github.com/analogdevicesinc/fru_tools"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/fru_tools/license.txt fru_tools
+
+package_table_items $((var++)) "pyadi-iio" " " "ADI BSD" "https://github.com/analogdevicesinc/pyadi-iio"
+wget https://raw.githubusercontent.com/analogdevicesinc/pyadi-iio/main/LICENSE
+html_pre_file LICENSE pyadi-iio
+rm LICENSE
+
+package_table_items $((var++)) "libm2k" " " "LGPL-2.1" "https://github.com/analogdevicesinc/libm2k"
+wget https://raw.githubusercontent.com/analogdevicesinc/libm2k/main/LICENSE
+html_pre_file LICENSE libm2k
+rm LICENSE
+
+package_table_items $((var++)) "iio-fm-radio" " " "GPL-2.0" "https://github.com/analogdevicesinc/iio-fm-radio"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/iio-fm-radio/license.txt iio-fm-radio
+
+package_table_items $((var++)) "jesd-eye-scan-gtk" " " "LGPL-2.1/GPL-2.0" "https://github.com/analogdevicesinc/jesd-eye-scan-gtk"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/jesd-eye-scan-gtk/LICENSE.txt jesd-eye-scan-gtk
+
+package_table_items $((var++)) "colorimeter" " " "LGPL-2.1/GPL-2.0" "https://github.com/analogdevicesinc/colorimeter"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/colorimeter/LICENSE.txt colorimeter
+
+package_table_items $((var++)) "gr-m2k" " " "GPL-3.0" "https://github.com/analogdevicesinc/gr-m2k"
+wget https://raw.githubusercontent.com/analogdevicesinc/gr-m2k/main/COPYING
+html_pre_file COPYING gr-m2k
+rm COPYING
+
+package_table_items $((var++)) "linux_image_ADI-scripts" " " "GPL-3.0" "https://github.com/analogdevicesinc/linux_image_ADI-scripts"
+html_pre_file ${ROOTFS_DIR}/usr/local/src/linux_image_ADI-scripts/LICENSE linux_image_ADI-scripts
```

Comment:
The same here - to see the list of branches used, check the table from: [https://confluence.analog.com/display/COS/2023_R2](https://confluence.analog.com/display/COS/2023_R2)